lib/testing-helpcenter/articles_with_tag_popup.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Methods to work with Pop up tags at https://helpcenter.onlyoffice.com/tags.aspx
+class ArticlesWithTagPopup
+  include PageObject
+
+  def initialize(instance)
+    super(instance.webdriver.driver)
+    @instance = instance
+    @pop_up_tags_xpath = "//div[@id = 'textHelpContent']//a"
+  end
+
+  def pop_up_tags_list
```

Comment:
It's a good idea to write simple documentation, especiflly types which this method return

Like

```
# @return [Array<String>] list of tags in popup window
```